code/modules/clothing/glasses/hud.dm

```diff
@@ -55,6 +55,30 @@
 			user.say("WHY IS THERE A BAR ON MY HEAD?!!")
 	return OXYLOSS
 
+/obj/item/clothing/glasses/hud/civ
+	name = "civilian HUD"
+	desc = "A heads-up display that shows only the current job ID of the user."
+	icon_state = "civ"
+	hud_type = DATA_HUD_SECURITY_BASIC
+	hud_trait = TRAIT_CIVILIAN_HUD
+
+/obj/item/clothing/glasses/hud/civ/night
```

Comment:
whats the point of this? i can just print an nvg sechud

Reply:
This is printable at all fabs, not just the security fab. Additionally, some security players (especially downstream on Skyrat) will have an issue with you running around with a secHUD.

Reply:
i agree, good for downstream.